### PR TITLE
Tio server

### PR DESCRIPTION
**Pull request by litongjava**
Please see readme.md

### CODE REVIEW

**Comment by litongjava** [comment]
All code has been completed and tested in the local environment, please review

**Comment by litongjava** [comment]
All code has been completed and tested in the local environment, please review again

**Comment by litongjava** [comment]
I've been submitting this for days, why is there no response?

**Comment by joanhey** [comment]
Read the issues for the delay
- #8622

### Review
Delete any file unnecessary for the bench:
- SQL file (hello_world.sql)
- Bin file in tests (test/java/新建日记本文档.jnt)
- ...

Happy new year !

File: frameworks/Java/tio-server/sql/hello_world.sql [hunk 1]
```diff
@@ -0,0 +1,76 @@
+/*
```
Comment (joanhey) [inline frameworks/Java/tio-server/sql/hello_world.sql:1]: Delete hello_world.sql, as it's an unnecessary extra file for the bench.

**Review by joanhey — CHANGES_REQUESTED** [review]
Please delete the binary in test/java, unnecesary for the bench.

**Comment by litongjava** [comment]
hello.when is the next test time?

**Comment by joanhey** [comment]
This PR don't pass the tests.
You need to fix it to pass the tests before is merged.

**Comment by litongjava** [comment]
removed,please check

**Comment by litongjava** [comment]
please approval

**Comment by litongjava** [comment]
fixed,please re-test

**Comment by litongjava** [comment]
fixed all problems, please approval

**Comment by litongjava** [comment]
@joanhey  please approval

**Comment by litongjava** [comment]
fixed,please check

**Comment by litongjava** [comment]
I've finish this pr, it can be merged

**Comment by litongjava** [comment]
@joanhey Hi! This PR has passed all checks and has no conflicts with the base branch. Could you please help review it? Thank you for your time!

File: frameworks/Java/tio-http-server/tio-server.dockerfile [hunk 2]
```diff
@@ -0,0 +1,19 @@
+FROM litongjava/maven:3.8.8-jdk8u391 AS builder
+WORKDIR /app
+
+COPY pom.xml pom.xml
+RUN mvn dependency:go-offline
```
Comment (joanhey) [inline frameworks/Java/tio-http-server/tio-server.dockerfile:5]: Please add `--quiet`
Now it's creating +5k useless lines in the logs.
(progress ....)
https://github.com/TechEmpower/FrameworkBenchmarks/actions/runs/11042481015/job/30945138563#step:9:5723

Comment (litongjava) [reply frameworks/Java/tio-http-server/tio-server.dockerfile]: @joanhey Thanks, I've fixed. Could you please help review it? Thank you for your time!

**Comment by litongjava** [comment]
@joanhey Thanks, I've fixed. Could you please help review it? Thank you for your time!

**Review by joanhey — APPROVED** [review]

**Pull request merged**